controllers/userController.js

```diff
@@ -52,7 +52,27 @@ const signIn = async (req, res) => {
     res.status(error.statusCode || 401).json({ message: error.message });
   }
 };
+
+const orderDetail = async function (req, res) {
+  try {
+    const userId = req.user;
+    const result = await userService.orderDetail(userId);
```

Comment:
주문 목록이기 때문에 detail보다는 list가 더 어울릴 것 같습니다.
```suggestion
    const result = await userService.getOrderList(userId);
```